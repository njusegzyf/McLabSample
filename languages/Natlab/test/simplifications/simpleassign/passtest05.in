
for i = foo()
    a(1) = foo();
end