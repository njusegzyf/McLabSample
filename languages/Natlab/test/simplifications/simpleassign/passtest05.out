
for i = foo()
    t1 = foo();
    a(1) = t1;
end
